.github/workflows/ci.yaml

```diff
@@ -50,5 +51,52 @@ jobs:
           docker pull gcr.io/oak-ci/oak:latest
           docker build --pull --cache-from=gcr.io/oak-ci/oak:latest --tag=gcr.io/oak-ci/oak:latest .
 
+      # Init .git repository used by check_generated.
+      # Workaround for https://github.com/GoogleCloudPlatform/cloud-builders/issues/236
+      - name: Git init
+        run: ./scripts/docker_run ./scripts/git_init
+
+      - name: Cargo crev
+        run: ./scripts/docker_run ./scripts/run_cargo_crev
+
       - name: Run command
         run: ./scripts/docker_run ./scripts/runner ${{ matrix.cmd }}
+
+      - name: Generate root CA certs
+        run: ./scripts/docker_run ./scripts/generate_root_ca_certs
+
+      # Check whether any of the previous steps resulted in file diffs that were not checked in or
+      # ignored by git.
+      - name: Git check diff
+        run: ./scripts/docker_run ./scripts/git_check_diff
```

Comment:
LG as long as this step actually works, please double check that it does fail if any auto-generated file does not match.

Reply:
Done